src/setupTests.tsx

```diff
@@ -2,8 +2,58 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
+import { Action, PreloadedState, ThunkAction } from '@reduxjs/toolkit';
 import '@testing-library/jest-dom';
-import { FullChannelMetadata, Record } from './app.types';
+import { Channel, FullChannelMetadata, Record, RecordRow } from './app.types';
+import { AppStore, RootState, setupStore } from './state/store';
+import { initialState as initialConfigState } from './state/slices/configSlice';
+import { initialState as initialColumnsState } from './state/slices/columnsSlice';
+import { initialState as initialSearchState } from './state/slices/searchSlice';
+import { render } from '@testing-library/react';
+import type { RenderOptions } from '@testing-library/react';
+import { Provider } from 'react-redux';
+
+export let actions: Action[] = [];
+export let resetActions = (): void => {
+  actions = [];
+};
+export const getState = (): RootState => ({
```

Comment:
It's a very minor thing but I might support this being renamed to `getInitialState` or `getInitialRootState`, purely because its current name could imply it will return the state as it currently is when called, when that's not actually the case.

Reply:
True, this is just copying from what it's called in SG/DG. Alternatively - `getMockState` or `mockGetState`?

Reply:
Up to you! Apologies for the spam comments on this thread, it started out as I only had one thing to say and then it just kept growing... still actively reviewing though!

Reply:
No worries - I tend to comment as I go along rather than comment all at once so I can't complain at it being done to me! I'll go with `getInitialState` as I think it works with the intention of what it's for more